src/elm/arithmetic.coffee

```diff
@@ -206,7 +206,12 @@ module.exports.MinValue = class MinValue extends Expression
 
   exec: (ctx) ->
     if MIN_VALUES[@valueType]
-      MIN_VALUES[@valueType]
+      if @valueType == '{urn:hl7-org:elm-types:r1}DateTime'
+        minDateTime = MIN_VALUES[@valueType].copy()
+        minDateTime.timezoneOffset = parseFloat(ctx.getTimezoneOffset())
```

Comment:
Why is `parseFloat` needed here (and in line 233 below)?  Are there use cases where `ctx.getTimezoneOffset()` might return a string?  I searched the rest of the codebase and as far as I can tell, we don't `parseFloat` offsets anywhere else.

Reply:
Good catch.

It was used because I noticed that in Bonnie, a string was being passed around:

![image](https://user-images.githubusercontent.com/14879344/52802449-f4d92f80-304d-11e9-9dc9-9f9241bf8ffb.png)

After some investigation, it looks like we are passing in '0' in Bonnie, where it should be 0.  I guess it was just working through the magic of JavaScript type conversion.

I'll take out the `ParseFloat` and modify Bonnie to properly pass in `0.0`.